code/game/objects/buckling.dm

```diff
@@ -22,9 +22,10 @@
 
 /atom/movable/MouseDrop_T(mob/living/M, mob/living/user)
 	. = ..()
-	if(can_buckle && istype(M) && istype(user))
-		if(user_buckle_mob(M, user))
-			return 1
+	if(!iscyborg(src)) //the code for buckling things to cyborgs is inside of robot.dmm, since cyborgs have different conditions for buckling things than most other buckle-able things do
+		if(can_buckle && istype(M) && istype(user))
+			if(user_buckle_mob(M, user))
+				return 1
```

Comment:
return true you noob

Reply:
1 is what this proc originally returned, but okay

Reply:
and thats why we have people like u to fix it